spec/requests/users_spec.rb

```diff
@@ -200,7 +200,7 @@
       # WST is not moderator by default, admin status is granted manually in
       # Discourse
       expect(answer_sso.moderator).to be false
-      expect(answer_sso.add_groups).to eq "wst,delegate"
+      expect(answer_sso.add_groups).to eq "delegate,wst"
```

Comment:
Might be cool to change the SSO interface to make sure the list of string tags (`wst`, `delegate`, etc.) is always sorted alohabetically. That way, we have a defined behavior of tests and don't need to worry about test chokes if some order of the rows in the DB changes.

Reply:
Okay done